podcasts/Onboarding/Models/PlusPricingInfoModel.swift

```diff
@@ -24,10 +24,12 @@ class PlusPricingInfoModel: ObservableObject {
 
         for product in products {
             let price = purchaseHandler.getPriceWithFrequency(for: product) ?? ""
+            let rawPrice = purchaseHandler.getPriceForIdentifier(identifier: product.rawValue)
```

Comment:
Initially I thought the `rawPrice` was returning a double value because of the "raw' what do you think about something like `displayPrice` instead?